### DISCUSSION TITLE
Insert link

### DISCUSSION THREAD

**Discussion by Darkxaser** [requarks/wiki#5264] [Help / Questions]
hello,

I'm testing your wiki right now and I have to say it's really good. But one thing I can not find, how can I insert it as here:

<img width="874" alt="Bildschirmfoto 2022-05-05 um 08 20 13" src="https://user-images.githubusercontent.com/100842056/166872832-d45be7cb-7db8-4700-9b28-9650952904d7.png">

can you please help me and tell me how I can insert this so?

many thanks

**Comment by Darkxaser** [comment]
i use the visual editor

**Reply by NGPixel** [reply]
This is only possible using the markdown editor.
